spec/features/managing_settings_spec.rb

```diff
@@ -1,19 +1,51 @@
+require 'spec_helper'
```

Comment:
We're moving to double quotes, there are lots of single quotes in the codebase from before the arrival of the Hound. Grandfathered exceptions aren't throwing errors because of rubocop_todo.yml. When I'm touching a file anyway, I often go through and get it up to snuff. It's too much to do all in one go, at least not without better test coverage.